docs/contributing/triaging-github-issues.md

```diff
@@ -63,6 +63,18 @@ It's nice to update labels as the state of an issue changes or if the type of an
 
 Check out [the docs on issue labeling for more info](/contributing/how-to-label-an-issue/)
 
+### Questions with immediate answers
+
+- Point to existing documentation to answer the question
+- If insufficient, do the following:
+  1. Provide an answer
+  2. Label the issue with documentation
+  3. Keep it open until a PR has added the answer to the documentation
```

Comment:
Thinking through the user workflow of looking for an answer and finding the issue, it would help to point to where it was documented. What do you think, does this make sense to add?
```suggestion
  3. Keep it open until a PR has added the answer to the documentation, and the issue includes a link to said documentation
```

Reply:
Yup, makes sense to me